Clarify support of multiple metrics in FAQ (#190)
DataDog/watermarkpodautoscaler: README.md

    Yes.

- **Does WPA support multiple metrics?**

>     Yes, WPA can scale on multiple metrics and works similar to [HPA](https://kubernetes.io/docs/tasks/run-application/horizontal-pod-autoscale/#scaling-on-multiple-metrics). WPA evaluates each metric separately and proposes maximum number of replicas of all.

Reviewer: ```suggestion
    Yes, WPA can scale on multiple metrics and works similar to [HPA](https://kubernetes.io/docs/tasks/run-application/horizontal-pod-autoscale/#scaling-on-multiple-metrics). WPA evaluates each metric separately and proposes the maximum number of replicas of all.
```
```suggestion
    Yes, WPA can scale on multiple metrics and works similar to [HPA](https://kubernetes.io/docs/tasks/run-application/horizontal-pod-autoscale/#scaling-on-multiple-metrics). WPA evaluates each metric separately and proposes the maximum number of replicas needed to support all metrics.
```

>     Yes, WPA can scale on multiple metrics and works similar to [HPA](https://kubernetes.io/docs/tasks/run-application/horizontal-pod-autoscale/#scaling-on-multiple-metrics). WPA evaluates each metric separately and proposes maximum number of replicas of all.

Reviewer: ```suggestion
    Yes, WPA can scale on multiple metrics and works similar to [HPA](https://kubernetes.io/docs/tasks/run-application/horizontal-pod-autoscale/#scaling-on-multiple-metrics). WPA evaluates each metric separately and proposes the maximum number of replicas needed to support all metrics.
```

Author: Thanks for the feedback! The idea about selecting maximum is that if WPA evaluates metric1, metric2, metric3 and for each it calculates 10, 20, 30 replica proposals respectively, the final proposal will be 30. Could 'support all metrics' be interpreted as 60 in this case?

Reviewer: Ah, I see. Perhaps this?

> WPA evaluates each metric separately and proposes the number of replicas associated with the metric that requires the largest number. For example, if WPA evaluates metric1, metric2, metric3 and for each it calculates 10, 20, 30 replica proposals respectively, the final proposal is 30.

>     Yes, WPA can scale on multiple metrics and works similar to [HPA](https://kubernetes.io/docs/tasks/run-application/horizontal-pod-autoscale/#scaling-on-multiple-metrics). WPA evaluates each metric separately and proposes maximum number of replicas of all.

Reviewer: ```suggestion
    Yes, WPA can scale on multiple metrics and works similar to [HPA](https://kubernetes.io/docs/tasks/run-application/horizontal-pod-autoscale/#scaling-on-multiple-metrics). WPA evaluates each metric separately and proposes the number of replicas associated with the metric that requires the largest number. For example, if WPA evaluates metric1, metric2, metric3, and for each it calculates 10, 20, 30 replica proposals respectively, the final proposal is 30.
```
